jobserver/models.py

```diff
@@ -120,6 +120,10 @@ def get_absolute_url(self):
     def get_zombify_url(self):
         return reverse("job-zombify", kwargs={"identifier": self.identifier})
 
+    @property
+    def is_finished(self):
```

Comment:
This is fine, but we've been using the term `completed` elsewhere in the job-runner so it might make sense to use that for consistency

Reply:
We're using `finished` for that consistently through job-server!  But that's becuase `completed_at` used to signify "successfully completed" and I wanted something that didn't signify a success state.  Wrote up a ticket (#363) to cover it so I don't forget.